Payments/Operation/05_ManageConfirmations.md

```diff
@@ -0,0 +1,123 @@
+[!!LIST (Confirmations](../UserInterface/06a_ListConfirmations.md)
+
+
+# Manage confirmations
+
+The *Confirmation* menu entry provides you with a view on all references to refunds that has been created. Before your company sends back the money to the customer, you have to check the refunds with this menu entry. This service is supported by the most payment service providers.
+> [Info] If a workflow decides that you have to sent back money to the customer, you have here the possibility to check the refunds before money is returned.   
+
+You can use this procedure to accept a refund so that the service payment provider gets the request to send back the money to the customer.    
+> [Info] The most payment service providers create a new reference, when a payment is to be refunded. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.   
+
+You can also reject a refund, if it is not justified, for example, because the goods have already been delivered. In case of faulty transactions you can void them to clear the database.   
+
+In the settings might have been applied special permissions for this menu entry.
+> [Info] You might not be able to access this menu entry, because you have no permissions to do it. Furthermore, you may have access, but you are not permitted to execute refunds above a certain amount (for example, more than 50€). 
+
+
+## Check confirmation transaction  
+
+*Payments > Confirmations > Tab LIST*  
+
+![Confirmation transaction](../../Assets/Screenshots/Payments/Confirmations/LISTConfirmations.png "[Confirmation transaction]")
+
+A confirmation transaction provides numerous payment-related details on a refund, which may vary depending on the payment service provider settings. To decide whether you can execute, reject or void a transaction, you can check its details.  
+> [Info] You can only change the status of a confirmation transaction, but not any data.
+
+
+#### Prerequisites 
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one refund has been created.
+- The view displays in minimum the *Amount* and the *Status* column. If it is necessary, it also shows the *Currency* column. For more information, see [LIST &ndash; Confirmations](../UserInterface/06_ListConfirmations.md#create-view).
+- You have the original reference ID for this transaction.
```

Comment:
Im Schritt Nr. steht "If desired", also nicht unbedingt notwendig. Ist das wirklich hier ein Prerequiste oder "nice to have"? Wenn man danach sucht/suchen möchte, muss man sie haben, daher würde ich es hier raus (evtl. als Info unter 1 packen evtl. mit der Info z.B., wo man diese Info finden kann).

Reply:
done